examples/ColumnGroupsExample.js

```diff
@@ -70,4 +70,4 @@ class ColumnGroupsExample extends React.Component {
   }
 }
 
-module.exports = ColumnGroupsExample;
+export default ColumnGroupsExample;
```

Comment:
https://github.com/Automattic/calypso-codemods

A few files had failed, either during the codemod run itself, or later on, because
* flow annotations messed up it's parsing
* some of our custom loaders load these files, and if we use export on them, it seems to fail. We might probably need to run Babel's transform on them
* I think some of the current scripts don't use babel-core or babel-registry, and since these don't get transpiled, it couldn't understand export syntax.

Reply:
Would it be possible to put the codemod changes and any other changes into separate PRs to make it easier to review the automated changes?

Reply:
yea, I'll close this one and do that.